kedro/templates/project/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/settings.py

```diff
@@ -1,40 +1,37 @@
-"""Project settings."""
+"""Project settings. There is no need to edit this file unless you want to change values
+from the Kedro defaults. For further information, including these default values, see
+https://kedro.readthedocs.io/en/stable/kedro_project_setup/settings.html."""
 
-# Instantiate and list your custom project hooks here
+# Instantiated project hooks.
 # from {{cookiecutter.python_package}}.hooks import ProjectHooks
 # HOOKS = (ProjectHooks(),)
 
-# List the installed plugins for which to disable auto-registry
+# Installed plugins for which to disable hook auto-registration.
 # DISABLE_HOOKS_FOR_PLUGINS = ("kedro-viz",)
 
-# Define where to store data from a KedroSession. Defaults to BaseSessionStore.
+# Class that manages storing KedroSession data.
 # from kedro.framework.session.store import ShelveStore
 # SESSION_STORE_CLASS = ShelveStore
-
-# Define keyword arguments to be passed to `SESSION_STORE_CLASS` constructor
+# Keyword arguments to pass to the `SESSION_STORE_CLASS` constructor.
 # SESSION_STORE_ARGS = {
 #     "path": "./sessions"
 # }
 
-# Define custom context class. Defaults to `KedroContext`
+# Class that manages Kedro's library components.
+# from kedro.framework.context import KedroContext
 # CONTEXT_CLASS = KedroContext
 
-# Define the configuration folder. Defaults to `conf`
+# Directory that holds configuration.
 # CONF_SOURCE = "conf"
 
-# Define the project ConfigLoader class here.
-# Defaults to kedro.config.ConfigLoader
+# Class that manages how configuration is loaded.
 # from kedro.config import TemplatedConfigLoader
 # CONFIG_LOADER_CLASS = TemplatedConfigLoader
-
-# Define keyword arguments to be passed to `CONFIG_LOADER_CLASS` constructor.
-# These kwargs depend on the `ConfigLoader` class implementation.
+# Keyword arguments to pass to the `CONFIG_LOADER_CLASS` constructor.
 # CONFIG_LOADER_ARGS = {
```

Comment:
Why don't we also have the catalog and parameter patterns here?

Reply:
Hmm yes, I suppose we could potentially make this configurable. It would need to work as an option for the `CONTEXT_CLASS` rather than `CONFIG_LOADER` I think, since that's where these patterns are actually used. Currently it is possible to do this by inheriting from `KedroContext` but it's a bit awkward since you need to overwrite some methods. We could make it easier by making these patterns class attributes or something.